src/paperless_tesseract/tests/test_parser.py

```diff
@@ -57,6 +57,20 @@ def test_get_text_from_pdf(self):
 
         self.assertContainsStrings(text.strip(), ["This is a test document."])
 
+    def test_get_pages_count(self):
```

Comment:
Though pretty clear, can you add the comments describing this test

Reply:
Yes, of course.

[Add comments to test_get_pages_count](https://github.com/paperless-ngx/paperless-ngx/pull/7750/commits/792e59355f0bfb10e6ed5af52b913b2b410338f7)

Reply:
The comments hes' referring to are of the format:

```py
        """
        GIVEN:
            - 
        WHEN:
            - 
        THEN:
            - 
        """
```

For example: https://github.com/paperless-ngx/paperless-ngx/blob/a9b7965dcf7129399242858fa29c0462e1100709/src/documents/tests/test_api_app_config.py#L23-L30

Reply:
Even if it's not for the API..?

Reply:
Yes, has nothing to do with API tests.

https://github.com/paperless-ngx/paperless-ngx/blob/a9b7965dcf7129399242858fa29c0462e1100709/src/paperless_tesseract/tests/test_parser.py#L324-L334

Reply:
Oh...ok
I think it would be good to add it to the CONTRIBUTING.md file.

Is that good for you ? [Add comments to test_get_pages_count](https://github.com/paperless-ngx/paperless-ngx/pull/7750/commits/a87db2b070eb3af9791ca1e837dd1ad6be3249ed)